.github/workflows/postman-collection-runner.yml

```diff
@@ -107,6 +107,11 @@ jobs:
               fi
           done
 
+      - name: Install newman from fork
+        run: |
+          npm install -g 'git+ssh://git@github.com:knutties/newman.git#feature/newman-dir'
+          npm config get prefix 
```

Comment:
Do we absolutely need the `npm config get prefix` here, or can we remove it?

Reply:
We can remove it - I kept it in to see what it outputs :)

Reply:
<img width="1322" alt="Screenshot 2023-08-24 at 8 08 01 AM" src="https://github.com/juspay/hyperswitch/assets/77204/743bbeee-5747-4baf-b5e1-4ba5691a56b4">

It is /usr/local

Reply:
> I kept it in to see what it outputs 

I figured that would be the case.